src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/value_axes.html

```diff
@@ -5,9 +5,10 @@
     </div>
 
     <button
-      aria-label="Add value axis"
+      data-test-subj="visualizeAddYAxisButton"
+      aria-label="Add Y axis"
```

Comment:
Can you confirm that we should use the term "Y axis" here instead of "value axis"?